web/html/src/manager/virtualization/guests/properties/guest-disks-panel.js

```diff
@@ -15,6 +15,7 @@ function getFileSourceFields(model: Object, index: number, pools: Array<Object>,
     if (model[`disk${index}_device`] === 'cdrom') {
       return [];
     }
+    const first_pool = pools.length > 0 ? pools[0].name : '';
```

Comment:
Just a question about this and all other evaluations added in this PR: would it be worth to check also if `pools` is not `null` and `pools[0].name` is not `null` as well, or would it be redundant because they can never be `null`?
Something like
```javascript
const first_pool = pools && pools.length > 0 && pools[0].name ? pools[0].name : '';
```

Reply:
This is ensured here: https://github.com/uyuni-project/uyuni/blob/master/web/html/src/manager/virtualization/guests/guest-properties.js#L86